src/stylesheets/base/_base.scss

```diff
@@ -27,10 +27,10 @@ body {
 .screen-reader {
   border: 0;
   clip: rect(0 0 0 0);
-  height: 1px;
-  margin: -1px;
+  height: .1rem;
+  margin: -.1rem;
   overflow: hidden;
   padding: 0;
   position: absolute;
-  width: 1px;
+  width: .1rem;
```

Comment:
Prettier is configured out of the box to add leading zeroes (so `0.1rem` versus `.1rem`). Have you looked into Prettier?

Reply:
I use Atom's Beautify. Will switch to Prettier if available in Atom!

Reply:
https://atom.io/packages/prettier-atom